tools/make/kube.mk

```diff
@@ -121,7 +121,8 @@ run-e2e: install-e2e-telemetry
 	kubectl wait --timeout=5m -n envoy-gateway-system deployment/envoy-gateway --for=condition=Available
 	kubectl apply -f test/config/gatewayclass.yaml
 ifeq ($(E2E_RUN_TEST),)
-	go test -v -tags e2e ./test/e2e --gateway-class=envoy-gateway --debug=true --cleanup-base-resources=$(E2E_CLEANUP)
+	go test -v -tags e2e ./test/e2e --gateway-class=envoy-gateway --debug=true
+	go test -v -tags e2e ./test/e2e/upgrade --gateway-class=envoy-gateway --debug=true --cleanup-base-resources=$(E2E_CLEANUP)
```

Comment:
should also add smiliar line on L128?

Reply:
yes indeed, so that we have an option to run just it.